src/alert.html

```diff
@@ -54,7 +54,7 @@ <h1 class="govuk-heading-xl alerts-icon__container alerts-icon__container--48">
       {{ alert_data.description | paragraphize }}
 
       <p class="govuk-body govuk-!-margin-bottom-0">
-        Issued by Cabinet Office <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
+        Issued by the UK government <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
```

Comment:
Is it `Issued by the UK government` or is it `Issued by UK Government`? I assume the former but worth checking with @karlchillmaid

Reply:
It should be `the`... also, I prefer `sent` to `issued`.

`Issued` doesn’t appear anywhere else in the site content.

Reply:
```suggestion
        Sent by the UK government <span class="relative-date__prefix">on </span><time class="relative-date" datetime="{{ alert_data.starts_date.as_iso8601 }}">{{ alert_data.starts_date.as_lang }}</time>
```